foundationdb/src/tuple/element.rs

```diff
@@ -256,7 +256,8 @@ impl Decode for String {
 }
 
 impl Encode for Vec<Element> {
-    fn encode<W: Write>(&self, w: &mut W) -> std::io::Result<()> {
+    fn encode<W: Write>(&self, w: &mut W, _in_tuple: bool) -> std::io::Result<()> {
+        // TODO: should this only write the Nested markers in the case of in_tuple?
```

Comment:
I think we only want to write the Nested values in the actual nested case, right? I didn't change this, but it seems like this is wrong, and should follow the same patter in the generated tuple macro.